src/js/structs/ServiceTree.js

```diff
@@ -168,21 +168,19 @@ module.exports = class ServiceTree extends Tree {
   }
 
   getServiceStatus() {
-    let {tasksRunning} = this.getTasksSummary();
-    let deployments = this.getDeployments();
-
-    if (deployments.length > 0) {
-      return ServiceStatus.DEPLOYING;
-    }
-
-    if (tasksRunning > 0) {
-      return ServiceStatus.RUNNING;
-    }
+    return this.reduceItems(function (statusSummary, item) {
```

Comment:
I'd rename the parameter to something as simple as `serviceStatus`, as we not  dealing with summary anymore.  #nit